docs/source/conf.py

```diff
@@ -46,7 +41,7 @@
 
 # General information about the project.
 project = "aiidalab-widgets-base"
-copyright_first_year = "2020"
+copyright_first_year = "2023"
```

Comment:
Shouldn't it be the year the documentation was created?

Reply:
You are correct, it didn't notice "first_year" :laughing: